contained-innersource-enables-collaborative-product-development.md

```diff
@@ -0,0 +1,35 @@
+## Title  
+Contained inner source enables collaborative product development
+
+## Context
```

Comment:
Georg, this is your scenario, so is your question rhetorical? 
I had gotten the impression when we were talking about this (in the dim sum place) that it was indeed the case that one BU was the current owner of the component. Is this the case?